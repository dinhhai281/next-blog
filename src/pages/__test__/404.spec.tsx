import React from 'react';
import NotFoundPage from '@app/pages/404';
import { render } from '@testing-library/react';

describe('404 Page', () => {
  describe('when render', () => {
    it('should match snapshot', () => {
      const { asFragment } = render(<NotFoundPage />);
      expect(asFragment()).toMatchSnapshot();
    });

    it('should contain text "404 Not Found"', () => {
      const { findByText } = render(<NotFoundPage />);
      expect(findByText('404 Not Found')).toBeTruthy();
    });
  });
});
