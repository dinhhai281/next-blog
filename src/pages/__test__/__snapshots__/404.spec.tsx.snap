// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page when render should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-0"
    >
      404 Not Found
    </p>
  </div>
</DocumentFragment>
`;
